api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -723,6 +695,20 @@ private boolean userHasEverLoggedIn(
     return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
   }
 
+  @AuthorityRequired(Authority.ACCESS_CONTROL_ADMIN)
+  @Override
+  public ResponseEntity<EmptyResponse> updateVerifiedInstitutionalAffiliation(
```

Comment:
I think ideally I'd run all updates like this directly through `updateProfile` or something like `adminUpdateProfile` rather than something specific to verified institution. I'll address that in RW-4980 where we'll take a holistic look at what we want the API structure to look like for the user admin

Reply:
Makes total sense – I'd love to brainstorm ideas here. As I'm going through this PR I'm starting to build some mental model of how profile vs. user APIs and models might be cleaned up for the future.

Reply:
Yes, let's talk about that later.